### ISSUE TITLE
`trybuild` coverage miss

### ISSUE THREAD

**Issue by wcampbell0x2a** [taiki-e/cargo-llvm-cov#299]
I see that `trybuild` _should_ generate a coverage hit, but I'm having trouble getting them to hit in this MR.

(The repo (for now) uses tarpaulin, but I might move to this crate)

# Expect

I expected the following  to have a hit: https://github.com/sharksforarms/deku/pull/352/files#diff-ccee6a304f8d0cb296ba83159aecd8534be0715ac4c0a3d0f1efb57e216ef490L191-L197. This should have been hit from the trybuild test @ `tests/test_compile/cases/ctx_validation.stderr`

```
$ TRYBUILD=overwrite cargo +nightly llvm-cov --workspace --doctests --release --html --ignore-filename-regex attributes
```

![image](https://github.com/taiki-e/cargo-llvm-cov/assets/15236002/301c872c-45c0-4089-a052-44db0501bf22)

**Comment by taiki-e** [comment]
Thanks for the report!

This appears to be due to a change in the target directory in trybuild 1.0.76 (https://github.com/dtolnay/trybuild/pull/219).
I can confirm that pinning the trybuild to 1.0.75 or older and running `cargo clean` and then cargo-llvm-cov solves the problem.

As for the change in trybuild 1.0.76, I thought I had addressed it in https://github.com/taiki-e/cargo-llvm-cov/pull/238, but that patch didn't seem to work correctly.

**Comment by taiki-e** [comment]
This is because I had missed the fact that the directory where the test workspace was generated had also been updated.

https://github.com/dtolnay/trybuild/pull/219/files#diff-1d06da761111802c793c6e5ca704bfa0d6336d0becf87fddff02d81548a838abL150

This will be fixed in #301.

**Comment by taiki-e** [comment]
Fixed in 0.5.26.

**Comment by wcampbell0x2a** [comment]
Thanks!
